### ISSUE TITLE
Upload revised forms and update guidance search

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#5846]
### Summary

**What we're after:**
Various forms are being updated as part of the Zip Code updating. When those forms are ready to be uploaded, we need to upload them and update guidance search. We also need to get Form 7 and its instructions into the guidance search.

### Related issues

#5818 

[How to guide for guidance search](https://docs.google.com/document/d/1hfvlYhGVNF0Km5vrAYtJak2cdAYLYQPbiPUfLxbRN1E/edit)

### Completion criteria

- [x] Revised forms are uploaded into the CMS (work with front end (John or Robert) to make sure they get in the correct place now that site map work is completed)
- [x] Guidance search is updated to add Form 7 and its instructions and to update time stamps for the other revised forms
- [x] Forms page is updated to remove the current link to Form 7and replace with separate links for Form 7 and Form 7 instructions and update link to Form 9 instructions
- [x] Redirect for old Form 9 instructions in place (filename changed)
- [X] Form 3P is straightened out. A changed filename was sent.
- [x] Other corrected forms were sent and uploaded.
- [x] Guidance search and redirects are updated for the corrected forms sent 9/21 and 9/22

**Comment by dorothyyeager** [comment]
Updated forms were sent (and then corrected forms sent again) on 9/21:
Form 1i
Form 1M
Form 1Mi
Form 3
Form 3X - typo in filename
Form 3P
Form 4
Form 5
Form 5i
Form 6
Form 7
Form 7i
Form 9i
Form 13i

Corrected forms sent late 9/21 or AM 9/22

Form 3P
Form 3X
Form 4i
Form 5
Form 9i

All have been uploaded. We corrected the filenames to make them fit our naming convention and are having Info verify we got them all. Then we'll update guidance search and redirects.
